cmd/pingpong/README.md

```diff
@@ -0,0 +1,61 @@
+# PingPong usage
+
+In order to run PingPong locally, an Algorand network must be available. It's
+sufficient to set it up using `goal network create -r {data directory} -r {network template json file}`.
```

Comment:
@algoidurovic Is it a typo or my misread?

```suggestion
sufficient to set it up using `goal network create -r {data directory} -t {network template json file}`.
```

Reply:
Good catch!